### ISSUE TITLE
Time costs problem when putting the data to RTX device using img.cuda()

### ISSUE THREAD

**Issue by Ruikun-Li** [pytorch/pytorch#31298]
## Issue description

I use pytorch and NVIDIA Graphics device to train deep learning models. But there are time cost problems in my experiments.
I use Dataloader to load the data, and put the data to graphics cards by using 'img = img.cuda()', when I use GTX1070(CUDA 9.0) and pytorch1.1, the step '.cuda()' costs about 0.4s. But when I use RTX2060S or RTX TITAX (CUDA 10.1) and pytorch1.3, the step '.cuda()' costs about 3s. That made my train time for each epoch increases so much(from 18s to 95s).
I already install the lasted driver for graphics cards. And I wonder if there is something wrong about RTX devices or pytorch?

## Code example

for index, (img, label) in enumerate(train_loader):

            time001 = time.time()
            if USE_GPU:
                img = img.cuda()
                label = label.cuda()

            print('Time:', time.time() - time001)

## System Info

- PyTorch:
- How you installed PyTorch (conda, pip, source): conda
- OS: Windows 10
- PyTorch version: 1.1 for GTX and 1.3.1 for RTX
- Python version: 3.6.9
- CUDA/cuDNN version: 9.0 for GTX and 10.1 for RTX
- GPU models and configuration: GTX 1070 and RTX 2060s/TITAN
- Versions of any other relevant libraries: Same for two experiments

**Comment by albanD** [comment]
Hi, please use the forum to ask questions: https://discuss.pytorch.org/

Also for cuda timing, because the api is asynchronous, you want to add `torch.cuda.synchronize()` before and after the call to make sure to measure only this call's runtime.
